bindata/network/ovn-kubernetes/self-hosted/multi-zone-interconnect/ovnkube-node.yaml

```diff
@@ -212,6 +246,24 @@ spec:
         - |
           #!/bin/bash
           set -euo pipefail
+
```

Comment:
We don't _need_ it, I just tried to avoid mixing a service account with cert credentials.
Do you think we should use a separate service account just for rbac-proxy?

Reply:
I think it is worth considering if otherwise it means we have to make this container privileged.

Reply:
I changed it so the service account is used for the rbac proxy stuff.